### ISSUE TITLE
seneca-amqp-transport is fragile for production that causes to denial-of-service 

### ISSUE THREAD

**Issue by thakerng** [senecajs/seneca-amqp-transport#64]
there are 2 services set up on different site, both connect to the same rabbitmq server I make the situation roughly here

```javascript
// service.js
seneca.add({role:'foo',cmd:'bar'},function(args,done){
   done(null,{message:"bar"});
});

seneca.add({role:'foo',cmd:'baz'},function(args,done){
     // done will not be call accidentally in this service
     ....
});

....
```

```javascript
// client.js

var app = express();

app.get('/api/bar',function(req,res){
   seneca.act({role:'foo',cmd:'bar'},function(err,resp){
     res.send(resp);
   })
});

app.get('/api/baz',function(req,res){
   seneca.act({role:'foo',cmd:'baz'},function(err,resp){
     res.send(resp);
   })
});

app.listen('3000',function(){
   console.log('listening 3000');
});

....
```

when I call a service `/api/bar` It's work well , but when I call `/api/baz` it's stuck because done was not called It's fine but I expect that when I call `/api/bar` again it should be work 
but it's stuck too and a message in Rabbitmq of `/api/bar` have not been consumed yet ,calling`/api/baz` causes a whole service definitely die I have to restart server to go on.

the situation above happens in seneca-amqp-transport but not in http and tcp 
Is there any suggestion to handle it? I think timeout is one of solution and important feature

**Comment by nfantone** [comment]
Unfortunately, as of `3.2.1`, timeout in Seneca is broken. Issue was fixed by [this PR](https://github.com/senecajs/gate-executor/pull/15), but it still needs to be rolled out. We are currently waiting for the next release of `gate-executor`.

In the meantime, one thing you can do is  _downgrade_ to Seneca `2.0.1`. It's a lame solution, but the best one I can give you as of now.

I'll try to push things around to get this updated, since its making an impact in people.

**Comment by nfantone** [comment]
This was fixed in `seneca@3.2.2`.
